lib/components/prometheusmetrics.js

```diff
@@ -60,7 +60,7 @@ function PrometheusMetrics() {
 
     // prom-client 9.0.0 has to be asked explicitly; previous versions would
     //   do this by default
-    if (client.collectDefaultMetrics) client.collectDefaultMetrics();
+    if (client.collectDefaultMetrics) {client.collectDefaultMetrics();}
```

Comment:
I guess we need to rule this, but I'd rather we did:

```
if (thing) {
 dodathing()
}
```

Reply:
```suggestion
    if (client.collectDefaultMetrics) {
        client.collectDefaultMetrics();
    }
```